pkgs/applications/editors/kakoune/plugins/build-kakoune-plugin.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, rtpPath ? "share/kak/autoload/plugins" }:
+rec {
+  buildKakounePlugin = attrs@{
+    name ? "${attrs.pname}-${attrs.version}",
+    namePrefix ? "kakplugin-",
+    src,
+    unpackPhase ? "",
+    configurePhase ? "",
+    buildPhase ? "",
+    preInstall ? "",
+    postInstall ? "",
+    path ? lib.getName name,
+    ...
+  }:
+    stdenv.mkDerivation (attrs // {
+      name = namePrefix + name;
+
+      inherit unpackPhase configurePhase buildPhase preInstall postInstall;
+
+      installPhase = ''
+        runHook preInstall
+
+        target=$out/${rtpPath}/${path}
+        mkdir -p $out/${rtpPath}
+        cp -r . $target
+
+        runHook postInstall
+      '';
+    });
+
+  buildKakounePluginFrom2Nix = attrs: buildKakounePlugin ({
+    buildPhase = ":";
+    configurePhase =":";
```

Comment:
```suggestion
    dontBuild = true;
    dontConfigure = true;
```

Reply:
I tried setting it to either `true` or `false`, but none of the works. Some plugins (e.g. [connect-kak](https://github.com/alexherbo2/connect.kak)) have a Makefile in the repository which then gets executed and fails.
Setting it to `"true"` seems to work. I left it unchanged at `":"`, which is consistent with the vim plugins.